mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/InvocationsAPI.cs

```diff
@@ -0,0 +1,367 @@
+using System;
```

Comment:
These changes are from:
https://github.com/mono/mono/pull/4164
They should not be committed.
Also, this pr needs to be refactored into normal commits with commit messages etc.